client/src/pages/Profiles/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { Typography, Box, Card, Rating, Divider } from '@mui/material';
+import useStyles from './useStyles';
+import StarIcon from '@mui/icons-material/Star';
+export default function ProfileCard({ profile }: any) {
+  const classes = useStyles();
+  const rating = 4;
+  const rates = ['$15', '$20', '$25', '$30'];
```

Comment:
Rather than using mock data, why don't we use the `price` from our Profile model